contents/page-sections/contact-us/form/index.tsx

```diff
@@ -0,0 +1,85 @@
+import { InstagramIcon, TwitterIcon, YoutubeIcon } from "@yamada-ui/lucide"
+import {
+  Button,
+  ButtonGroup,
+  Container,
+  FormControl,
+  Heading,
+  HStack,
+  IconButton,
+  Input,
+  Text,
+  Textarea,
+  VStack,
+} from "@yamada-ui/react"
+import type { FC } from "react"
+import { contacts } from "./contacts"
+
+const Form: FC = () => {
+  return (
+    <Container
+      bgGradient="linear(to-r, primary.700, primary.500)"
+      flexDir={{ base: "row", md: "column" }}
+      p="2xl"
```

Comment:
It is easier to see if you change the `padding` between `sp` and `md`.

```suggestion
      p={{base: "2xl", md: "lg", sm: "md"}}
```

Reply:
Fixed.